monocle/db.py

```diff
@@ -225,6 +225,8 @@ class Sighting(Base):
     sta_iv = Column(TINY_TYPE)
     move_1 = Column(SmallInteger)
     move_2 = Column(SmallInteger)
+    cp = Column(SmallInteger)
+    cp_multiplier = Column(FLOAT_TYPE)
```

Comment:
I think it would be better to save the level here.
That way `cp_multiplier` => `level` is computed once, otherwise you are doing it once for **every** visitor (in `web.py` / `web_sanic.py`) besides an extra one for the webhook.
besides that a `SmallInteger` should use less space than a `FLOAT_TYPE`